.github/workflows/on_pull_request.yml

```diff
@@ -0,0 +1,60 @@
+name: Build & Deploy
+on: [pull_request]
+
+env:
+  CONTAINER_REGISTRY: ghcr.io
+  CONTAINER_REGISTRY_USER: ${{ secrets.GHCR_CONTAINER_REGISTRY_USER }}
+  CONTAINER_REGISTRY_PASSWORD: ${{ secrets.GHCR_TOKEN }}
+  CONTAINER_REGISTRY_REPO: ghcr.io/city-of-helsinki/${{ github.event.repository.name }}
+  KUBECONFIG_RAW: ${{ secrets.KUBECONFIG_RAW }}
+  BUILD_ARTIFACT_FOLDER: "build_artifacts"
+  SERVICE_ARTIFACT_FOLDER: "service_artifacts"
+  # Default usernames and passwords for postgres setups
+  # Note: These are the credentials that will be used for testing
+  DATABASE_USER: user
+  DATABASE_PASSWORD: testing-password
+  K8S_CLUSTER_ISSUER: "certificate-letsencrypt-prod"
+  K8S_SECRET_SECRET_KEY: ${{ secrets.SECRET_KEY }}
+  APP_MIGRATE_COMMAND: /app/.prod/on_deploy.sh
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        uses: andersinno/kolga-build-action@v2
+        with:
+          disable_cache: true
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: build_artifacts
+          path: build_artifacts/
+
+  review:
+    runs-on: ubuntu-latest
+    needs: build
+    name: Review
+    steps:
+      - uses: actions/checkout@v2
+      - uses: andersinno/kolga-setup-action@v2
+
+      - name: Review-Services
+        uses: andersinno/kolga-deploy-service-action@v2
+        with:
+          projects: CITY_OF_HELSINKI_AHTI
+        env:
+          ENVIRONMENT_URL: https://${{ env.K8S_NAMESPACE }}.newtest.kuva.hel.ninja
+          POSTGRES_IMAGE: "docker.io/andersinnovations/postgis:11-bitnami"
+
+      - name: Deploy
+        uses: andersinno/kolga-deploy-action@v2
+        env:
```

Comment:
Application specific variables could be groupped and introduced after the mandatory ones (like ENVIRONMENT_URL)